docs/en/drivers/ios-xcuitest-real-devices.md

```diff
@@ -126,8 +126,13 @@ installed, and is _not_ recommended):
     $ which appium
     /path/where/installed/bin/appium
 ```
+```
+    From appium v1.15, WebDriverAgent.bundle will be created automatically in Resources directory under 
```

Comment:
Since Appium 1.15 ... is created automatically...
Proceed with the tutorial below if Appium version is lower than 1.15